pkgs/development/interpreters/wamr/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wamr";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "bytecodealliance";
+    repo = "wasm-micro-runtime";
+    rev = "WAMR-${version}";
+    hash = "sha256-jpT42up9HAVJpo03cFrffQQk2JiHEAEepBGlU4RUfNU=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  sourceRoot = "source/product-mini/platforms/linux";
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp iwasm $out/bin
```

Comment:
You could collapse this into one line if you wanted:

```sh
install -Dt $out/bin iwasm
```

But it doesn't really matter either way — I'll merge once OfBorg is passing regardless.

Reply:
I'll add your suggestion along with the release. Thanks.